lib/class-vip-request-block.php

```diff
@@ -128,12 +128,17 @@ public static function block_and_log( string $value, string $criteria ) {
 			newrelic_ignore_transaction();
 		}
 
+		if ( defined( 'WP_CLI' ) && constant( 'WP_CLI' ) ) {
```

Comment:
https://make.wordpress.org/cli/handbook/guides/commands-cookbook/#include-in-a-plugin-or-theme

```suggestion
		if ( defined( 'WP_CLI' ) && WP_CLI ) {
```

Reply:
we've been using `constant` for quite a while because it's better testable that way.

Reply:
Ah, TIL. Thanks! Should we suggest an update to WP-CLI docs?